### DISCUSSION TITLE
unity merge할때 나오는 수많은 comflict들의 대처법...

### DISCUSSION THREAD

**Discussion by hongsam14** [hongsam14/myCPU#32] [Q&A]
42 동료와 함께 유니티를 활용한 팀 프로젝트를 하나 열어서 초반 궤도에 진입했습니다. 하지만 혼자 할땐 몰랐던 문제에 직면했는데
여러 comflict들이 생겼다는 것이고 그로인해 수백개의 오류가 생긴다는 것입니다...

**Comment by jongfeel** [comment]
어떤 문제가 생기는지 얘기해 보고 해결해 나가 보죠

**Comment by hongsam14** [comment]
서로 json 파일에서 comflict가 난다고 경고가 났는데 실수로 merge를 해버렸습니다. 그래서 head <<<<이런 메시지들이 삽입되면서 유니티에서 에러가 나는 것 같습니다. 협업하며 
유니티의 버전과 패키지들을 어떻게 관리해야할지 감이 안잡히는 것 같습니다.
